stdlib/types.pyi

```diff
@@ -236,15 +236,15 @@ class AsyncGeneratorType(AsyncGenerator[_T_co, _T_contra]):
     __name__: str
     __qualname__: str
     def __aiter__(self) -> AsyncGeneratorType[_T_co, _T_contra]: ...
-    async def __anext__(self) -> _T_co: ...
-    async def asend(self, __val: _T_contra) -> _T_co: ...
+    def __anext__(self) -> Coroutine[Any, Any, _T_co]: ...
```

Comment:
Out of curiosity, what is the reason for these changes?

Reply:
Well,
```
>>> inspect.iscoroutinefunction(types.AsyncGeneratorType.aclose)
False
```
I'm not sure if this matters at all, cc @AlexWaygood who added this to stubtest.

Reply:
cc. @AlexWaygood who made them async only a few weeks ago as well...

Reply:
"Classes" in the `types` module are weird, I'm happy to go with whatever stubtest thinks best personally.

Reply:
(the other big change we've made recently is switching a lot of `(...) -> Awaitable` into `async (...) -> ` which actually does make a difference to type checking)

Reply:
Interestingly, it looks like there are already at least 2 open BPO issues about the fact that `inspect.iscoroutinefunction` returns `False` for `types.AsyncGeneratorType.__anext__`, `.asend` and `.aclose`. https://bugs.python.org/issue42760.

I'm not really sure what we should make of that, but it's interesting!